client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -98,6 +100,7 @@ function ReaderSubscriptionListItem( {
 					className="reader-subscription-list-item__site-url"
 				>
 					{ formatUrlForDisplay( siteUrl ) }
+					{ feed && feed.last_update && ( ' ' + translate( 'updated' ) + ' ' + lastUpdatedDate ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Updated!' )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/last-updated-date-som)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).